### PR TITLE
Service endpoint Config tool

### PR DESCRIPTION
**Pull request by vsaurab10**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +37,27 @@ After deploying Azure resource from ARM template, follow the below steps to comp
 
 Please refer to the instructions [here](https://docs.microsoft.com/en-us/dynamics365/field-service/cfs-authorize-api-connection). This is required to begin using Connected Field Service with IoT Hub.
 
+### Configure Service Endpoint
+
+In order to enable the communication between your CRM organization and IoT Hub, configuring the Service Endpoint in CRM organization is mandatory. You can configure it by running the ConfigureIoTEndpoint tool which is present in the repository [here](). Please follow the instructions below to run the tool.
+
+1. Downlod tool's zip file and extract it on your local machine
```
Comment (nickrm101) [inline README.md:44]: Download* (typo)

File: README.md [hunk 2]
```diff
@@ -34,6 +37,27 @@ After deploying Azure resource from ARM template, follow the below steps to comp
...
+
+   - XXX - Username used for logging in to your online D365 organization
+   - YYY - Password (in quotes) used for logging in to your online D365 organization
+   - ZZZ - Service Bus name. You can find it by logging in to your Azure portal and navigating to the Resource Group where your resources are deployed. From the Resources tab open resource of type Service Bus Namespace. Click on three dots (...) and select Copy title to Clipboard
+   - AAA - Service Bus Key. On the same page, in left menu, under Settings click on Shared Access Policies > RootManageSharedAccessKey > Primary Key. Copy the Primary Key value.
+   - BBB - Queue Name in Service Bus. Open the resource of type Service Bus Namespace. In left menu, under Entities click on Queues. Copy the queue name ending with "-crm"
+   - CCC - Your D365 organization's fulle name without "https://"
+
+   Example:
+   ConfigureIoTEndpoint.exe -u contoso@yourcrmtenant.onmicrosoft.com -p xxxxxx -s sukrisdemo6388cc05a2f544d08b7c901ef02b563a -k xxxxxxxxxxxxxxxxx -q sukrisdemo-crm -d yourorgname.crm.dynamics.com
```
Comment (nickrm101) [inline README.md:59]: Change to remove alias

**Review by nickrm101 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (vsaurab10) [reply README.md]: Removed

File: README.md (hunk 1 above)
Comment (vsaurab10) [reply README.md]: Updated

**Review by nickrm101 — APPROVED** [review]

**Pull request closed without merging**